schema.sql

```diff
@@ -1,5 +1,23 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
+    id INT GENERATED ALWAYS AS IDENTITY,
+     animal_name TEXT,
+     DATE_OF_BIRTH DATE,
+     ESCAPE_ATTEMPTS INT,
+     NEUTERED BIT,
+     WEIGHT_KG DECIMAL,
+     PRIMARY KEY (id)
 );
+
+ALTER TABLE animals ADD species varchar(250);
+
+
+CREATE TABLE owners (id INT GENERATED ALWAYS AS IDENTITY, FULL_NAME VARCHAR(250), AGE INT, PRIMARY KEY (id));
+CREATE TABLE species (id INT GENERATED ALWAYS AS IDENTITY, NAME VARCHAR (250), PRIMARY KEY (id));
+ALTER TABLE animals DROP COLUMN species;
+ALTER TABLE animals ADD species_id INT;
+ALTER TABLE animals ADD CONSTRAINT fk_species FOREIGN KEY (species_id) REFERENCES species(ID);
+ALTER TABLE animals ADD CONSTRAINT fk_owners FOREIGN KEY (owner_id) REFERENCES owners(ID);
+CREATE TABLE vets ( id INT GENERATED ALWAYS AS IDENTITY, NAME VARCHAR(250), AGE INT, DATE_OF_GRADUATION DATE, PRIMARY KEY (id) );
+CREATE TABLE visits ( NAME VARCHAR(250), ANIMAL_NAME VARCHAR (250), DATE_OF_VISIT DATE );
```

Comment:
- [ ] There should be a many-to-many relationship between the tables `animals` and `vets`: an animal can visit multiple vets and one vet can be visited by multiple animals. Kindly, make sure the join table `visits` handle this relationship 👍🏻 
- Your code might look like this:
 ```sql
   CREATE TABLE visits (
   animal_id int,
   vet_id int,
   visit_date date -- another name is acceptable
   );
```

Reply:
> * [ ]  There should be a many-to-many relationship between the tables `animals` and `vets`: an animal can visit multiple vets and one vet can be visited by multiple animals. Kindly, make sure the join table `visits` handle this relationship 👍🏻
> * Your code might look like this:
> 
> ```sql
>   CREATE TABLE visits (
>   animal_id int,
>   vet_id int,
>   visit_date date -- another name is acceptable
>   );
> ```

- [ ] Kindly make sure you address this 🙃